spring-boot-project/spring-boot-test/src/test/java/org/springframework/boot/test/system/OutputCaptureRuleTests.java

```diff
@@ -37,10 +37,27 @@ public void toStringShouldReturnAllCapturedOutput() {
 		assertThat(this.output.toString()).contains("Hello World");
 	}
 
+	@Test
+	public void getAllShouldReturnAllCapturedOutput() {
+		System.out.println("Hello World");
```

Comment:
This test should generate output using both `System.err` and `System.out` and then check that `getAll()` contains all of it.